### ISSUE TITLE
Grafana deployment error

### ISSUE THREAD

**Issue by akoutmos** [istio/istio#10273]
**Describe the bug**
I am trying to install Istio 1.0.4 via Helm on my DigitalOcean Kubernetes cluster and the installation is failing because of a failure when trying to configure Grafana. This is the error that I receive"
`Error: release istio failed: Deployment.apps "grafana" is invalid: spec.template.spec.containers[0].volumeMounts[1].name: Not found: "dashboards-istio"`

**Expected behavior**
I expected that the installation would leverage the DigitalOcean storage class that it creates and store the packaged dashboards there.

**Steps to reproduce the bug**
Follow the installation steps provided below

**Version**
Istio Version: 1.0.4
Kubernetes Version: 1.12.3

**Installation**
```bash
$ kubectl --kubeconfig="$(KUBECONFIG)" apply -f install/kubernetes/helm/helm-service-account.yaml

$ helm --kubeconfig $(KUBECONFIG) init --service-account tiller

$ helm --kubeconfig $(KUBECONFIG) install install/kubernetes/helm/istio \
--name istio \
--namespace istio-system \
--set security.enabled=true \                                                             
--set grafana.enabled=true \                                                              
--set grafana.storageClassName=do-block-storage \                                         
--set grafana.persist=true \                                                              
--set prometheus.enabled=true \                                                           
--set servicegraph.enabled=true \                                                         
--set tracing.enabled=true \                                                              
--set kiali.enabled=true
```

**Environment**
Cloud Vendor: DigitalOcean

Any help would be much appreciated. Thanks!

**Comment by mk01** [comment]
@akoutmos 

this is a typo in 1.0.4 in deployment.yaml of grafana (install/kubernetes/helm/istio/charts/grafana/templates/deployment.yaml).
change the last 6 lines from:
```
{{- else }}
        emptyDir: {}
      - name: dashboards-istio
        configMap:
          name:  istio-grafana-configuration-dashboards
{{- end }}
```

to 

```
{{- else }}
        emptyDir: {}
{{- end }}
      - name: dashboards-istio
        configMap:
          name:  istio-grafana-configuration-dashboards
```

**Comment by akoutmos** [comment]
Thanks for the response! I'll give that a whirl and report back

**Comment by linsun** [comment]
just checked, this issue is fixed in 1.1.

**Comment by linsun** [comment]
given we don't have a plan to spin 1.0.x, close this.  pls reopen if it is not fixed in 1.1
